Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift

```diff
@@ -287,14 +288,14 @@ extension DaysUntilBirthdayUITests_iOS {
   }
 
   /// This method looks for an account in the current view that reflects an already-signed-in state, and taps it.
-  /// Returns true if the signed-in account was found and tapped, otherwise returns false.
+  /// - returns: true if the signed-in account was found and tapped, otherwise returns false.
   func findAndTapExistingSignedInEmail() -> Bool {
     guard sampleApp.staticTexts[Credential.email.rawValue].exists else {
-      XCTFail("Email used for previous sign-in not in list")
+      XCTFail(String(format: "Could not find %@ in list of previously-used accounts.", Credential.email.rawValue))
```

Comment:
Apologies -- I think I suggested that we interpolate the email in here. Let's not print that out.